packages/@uppy/companion-client/src/RequestClient.js

```diff
@@ -167,9 +168,24 @@ export default class RequestClient {
     }
   }
 
-  async get (path, skipPostResponse) { return this.#request({ path, skipPostResponse }) }
+  async get (path, options = undefined) {
+    // TODO: remove boolean support for options that was added for backward compatibility.
+    // eslint-disable-next-line no-param-reassign
+    if (typeof options === 'boolean') options = { skipPostResponse: options }
+    return this.#request({ ...options, path })
```

Comment:
I find spreading undefined always weird, even though it's possible. But I remember I crashed IE11 once because I spreaded undefined so might as well make the default an empty object?

Reply:
No, there's no need to create a throw away object, IE is no longer an acceptable reason IMO.